### PR TITLE
[#4238] fix (trino-connector) Make the Trino connector test more stable and improve the logs

### PR DESCRIPTION
**Pull request by diqiu50**
### What changes were proposed in this pull request?

1. Add more output log  for run the Trino Integration test.
2. Enable debug for TrinoQuery runner
3. Increase timeout for wait catalog to create.

### Why are the changes needed?

Fix: #4238

### Does this PR introduce _any_ user-facing change?

NO

### How was this patch tested?

NO

### CODE REVIEW

File: integration-test/src/test/java/org/apache/gravitino/integration/test/trino/TrinoQueryRunner.java [hunk 1]
```diff
@@ -61,7 +61,7 @@ private QueryRunner createQueryRunner() throws Exception {
             .timeZone(ZoneId.systemDefault())
             .clientRequestTimeout(new Duration(30, TimeUnit.SECONDS))
             .build();
-    return new QueryRunner(trinoUri, session, false, HttpLoggingInterceptor.Level.NONE);
+    return new QueryRunner(trinoUri, session, true, HttpLoggingInterceptor.Level.NONE);
```
Comment (diqiu50) [inline integration-test/src/test/java/org/apache/gravitino/integration/test/trino/TrinoQueryRunner.java:64]: Enable debug of QueryRunner

**Comment by jerryshao** [comment]
@yuqi1129 can you please help to review?

File: trino-connector/src/main/java/org/apache/gravitino/trino/connector/system/storedprocdure/AlterCatalogStoredProcedure.java [hunk 2]
```diff
@@ -140,7 +141,7 @@ public void alterCatalog(
     } catch (Exception e) {
       throw new TrinoException(
           GravitinoErrorCode.GRAVITINO_UNSUPPORTED_OPERATION,
-          "alter catalog failed. " + e.getMessage(),
+          "alter catalog failed. " + (StringUtils.isEmpty(e.getMessage()) ? "" : e.getMessage()),
```
Comment (yuqi1129) [inline trino-connector/src/main/java/org/apache/gravitino/trino/connector/system/storedprocdure/AlterCatalogStoredProcedure.java:144]: I'm not very clear about the changes here,  if `e.getMessage()` is empty or null, the code changes seems be to the same as before.

File: trino-connector/src/main/java/org/apache/gravitino/trino/connector/catalog/CatalogConnectorManager.java [hunk 3]
```diff
@@ -60,7 +60,7 @@ public class CatalogConnectorManager {
 
   private static final int CATALOG_LOAD_FREQUENCY_SECOND = 10;
   private static final int NUMBER_EXECUTOR_THREAD = 1;
-  private static final int LOAD_METALAKE_TIMEOUT = 30;
+  private static final int LOAD_METALAKE_TIMEOUT = 60;
```
Comment (yuqi1129) [inline trino-connector/src/main/java/org/apache/gravitino/trino/connector/catalog/CatalogConnectorManager.java:63]: Why do we increase this value? Is it really take so long to load metalake?

Comment (diqiu50) [reply trino-connector/src/main/java/org/apache/gravitino/trino/connector/catalog/CatalogConnectorManager.java:63]: In distribution environment.  if found create catalog it's take too long time. the function throw the timeout exception. I try to increase this value, and add some log to check the problems

File: trino-connector/src/main/java/org/apache/gravitino/trino/connector/system/storedprocdure/AlterCatalogStoredProcedure.java (hunk 2 above)
Comment (diqiu50) [reply trino-connector/src/main/java/org/apache/gravitino/trino/connector/system/storedprocdure/AlterCatalogStoredProcedure.java:144]: if e.getMessage() == null. the error message is `alter catalog failed. null`. so it change it.

**Review by yuqi1129 — APPROVED** [review]
LGTM

**Pull request merged**